packages/react-core/src/demos/Chart/PrometheusChart.md

```diff
@@ -0,0 +1,207 @@
+      labels={({ datum }) => `${datum.y !== null ? datum.y : 'no data'}`}
+      labelComponent={
+        <ChartLegendTooltip legendData={legendData} title={datum => dateTimeFormatterWithSeconds.format(datum.x)} />
+      }
+      mouseFollowTooltips
+      voronoiDimension="x"
+      voronoiPadding={50}
+    />
+  );
+
+  const theme = getCustomTheme(ChartThemeColor.multiUnordered, ChartThemeVariant.light);
+  const colors = theme.line.colorScale;
+
+  const xAxisTickFormat = d => {
+    return timeFormatter.format(d);
+  };
+
+  const GroupComponent = isStacked ? ChartStack : ChartGroup;
+  const ChartComponent = isStacked ? ChartArea : ChartLine;
+
+  const PageToolbar = <div>need to implement new toolbar</div>;
+  const Header = (
+    <PageHeader
+      logo={<Brand src={imgBrand} alt="Patternfly Logo" />}
+      toolbar={PageToolbar}
+      avatar={<Avatar src={imgAvatar} alt="Avatar image" />}
+    />
+  );
+  const pageId = 'main-content-page-layout-default-nav';
+
+  return (
+    <React.Fragment>
+      <Page header={Header} mainContainerId={pageId}>
+        <PageSection variant={PageSectionVariants.light}>
+          <TextContent>
+            <Text component="h1">Prometheus Chart</Text>
+            <Text component="p">
+              This demo mainly shows how to format data from the Prometheus database and use the PatternFly Chart to
+              consume the data. This demo could be used on monitoring visualization.
+            </Text>
+          </TextContent>
+        </PageSection>
+        <Divider component="div" />
+        <PageSection variant={PageSectionVariants.light}>
+          <div ref={containerRef}>
+            <Checkbox id="stacked" isChecked={isStacked} label="Stacked" onChange={v => setIsStacked(v)} />
+            <Chart
+              containerComponent={graphContainer}
+              ariaTitle="query browser chart"
+              domainPadding={{ y: 1 }}
+              height={400}
+              padding={{
+                bottom: 75,
+                left: 50,
+                right: 50,
+                top: 50
+              }}
+              legendData={legendData}
+              legendPosition="bottom-left"
+              scale={{ x: 'time', y: 'linear' }}
+              theme={theme}
+              width={width}
+            >
+              <ChartAxis fixLabelOverlap={true} tickCount={10} tickFormat={xAxisTickFormat} />
+              <ChartAxis crossAxis={false} dependentAxis showGrid tickCount={6} />
+              <GroupComponent>
+                {data.map((series, i) => {
+                  const [metric, values] = series;
+                  if (values === null) {
+                    return null;
+                  }
+                  const color = colors[i % colors.length];
+                  const style = {
+                    data: { [isStacked ? 'fill' : 'stroke']: color },
+                    labels: { fill: color, name: tooltipSeriesNames[i] }
+                  };
```

Comment:
You shouldn't need to define the color style, unless overriding colors. The chart will iterate over the color scale defined by the multiUnordered theme.